Content.Server/Chemistry/TileReactions/SpillTileReaction.cs

```diff
@@ -37,6 +37,11 @@ public FixedPoint2 TileReact(TileRef tile, ReagentPrototype reagent, FixedPoint2
                 var step = entityManager.EnsureComponent<StepTriggerComponent>(puddleUid);
                 entityManager.EntitySysManager.GetEntitySystem<StepTriggerSystem>().SetRequiredTriggerSpeed(puddleUid, _requiredSlipSpeed, step);
 
+                var slow = entityManager.EnsureComponent<SlowContactsComponent>(puddleUid);
+                var speedModifier = 1 - reagent.Viscosity;
+                slow.WalkSpeedModifier = speedModifier;
+                slow.SprintSpeedModifier = speedModifier;
+
```

Comment:
Doesn't this need to be dirtied otherwise it's going to mispredict, also does movespeed need refreshing?

Reply:
no, it's being refreshed every frame in update loop
![image](https://github.com/space-wizards/space-station-14/assets/40753025/dca27b70-48f1-4eba-b519-e2483764f2a9)